index.html

```diff
@@ -650,88 +682,118 @@ <h3>JSON Examples</h3>
   }
 }
       </pre>
+</section>
 
-      <pre class="example" title="A verifiable credential controller document">
+<section>
+  <h2>
+    Presentations
+  </h2>
+  
+<pre class="example" title="Credential">
```

Comment:
```suggestion
<pre class="example" title="A Verifiable Presentation Containing Many Verifiable Credentials">
```

Reply:
Good suggestion, but it does not contain them... it contains one and references many, can you amend your suggestion to "A Verifiable Presentation referencing and containing Verifiable Credentials" ? or something similar?